src/vma/proto/dst_entry.cpp

```diff
@@ -623,12 +624,11 @@ void dst_entry::do_ring_migration(lock_base& socket_lock)
 	resource_allocation_key *new_key = m_ring_alloc_logic.get_key();
 	uint64_t new_calc_id = m_ring_alloc_logic.calc_res_key_by_logic();
 	// Check again if migration is needed before migration
-	if (new_key->get_user_id_key() == new_calc_id) {
+	if (old_key.get_user_id_key() == new_calc_id &&
```

Comment:
Please set new key below m_ring_alloc_logic.calc_res_key_by_logic().
It's very confusing...
Also for rx.

Reply:
Done.